grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -72,25 +73,25 @@ class ScoreL2RPN2023(ScoreL2RPN2020):
     """
 
     def __init__(
-        self,
-        env,
-        env_seeds=None,
-        agent_seeds=None,
-        nb_scenario=16,
-        min_losses_ratio=0.8,
-        verbose=0,
-        max_step=-1,
-        nb_process_stats=1,
-        scale_assistant_score=100.0,
-        scale_nres_score=100.,
-        weight_op_score=0.6,
-        weight_assistant_score=0.25,
-        weight_nres_score=0.15,
-        weight_confidence_assistant_score=0.7,
-        min_nres_score=-100,
-        add_nb_highres_sim=False,
+            self,
+            env,
+            env_seeds=None,
+            agent_seeds=None,
+            nb_scenario=16,
+            min_losses_ratio=0.8,
+            verbose=0,
+            max_step=-1,
+            nb_process_stats=1,
+            scale_assistant_score=100.0,
+            scale_nres_score=100.,
+            weight_op_score=0.6,
+            weight_assistant_score=0.25,
+            weight_nres_score=0.15,
+            # weight_confidence_assistant_score=0.7,
```

Comment:
This has been commented, we should maybe juste remove it (we no longer have two components in the assistant score).